Client/Frontend/Fakespot/Client/ProductAnalysisData.swift

```diff
@@ -20,6 +20,14 @@ struct ProductAnalysisData: Codable {
         case analysisUrl = "analysis_url"
         case highlights
     }
+
+    var cannotBeAnalyzedCardVisible: Bool {
+        productId == nil && needsAnalysis == true
```

Comment:
nit:  It's already a Boolean expression, so there's no need to explicitly compare it to true.

Reply:
Yes, the problem is they are optional and need to be unwrapped first to work. So something like 
`if let needsAnalysis, needsAnalysis {}` which is not ideal.